packages/mds-orm/tests/audits.ts

```diff
@@ -26,6 +26,7 @@ export default () =>
           .toUpperCase()}-${index.toString().padStart(6, '0')}`,
         provider_device_id: index % 2 === 0 ? uuid() : undefined,
         timestamp: recorded,
+        id: 0,
```

Comment:
I would not set a value...

Reply:
It gets mad if I don't though. Suggestions for getting around this?